projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py

```diff
@@ -56,7 +56,7 @@ def __init__(self):
         self.TRAIN_GPU_IDS = [0, 1, 2, 3, 4, 5, 6]
         self.VALID_GPU_IDS = [7]
         self.TEST_GPU_IDS = [7]
-        self.ADVANCE_SCENE_ROLLOUT_PERIOD = 10 ** 13
+        self.ADVANCE_SCENE_ROLLOUT_PERIOD = None
```

Comment:
Why did you change this? (To be honest I just inherited this value from older configs I saw and never touched it, I am just curious as to why we are changing it now)

Reply:
I think this was inherited from a made-in-Jordi hack :D I much prefer having None...